src/App.js

```diff
@@ -1,29 +1,32 @@
 import React from 'react';
-
 import './App.css';
 
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
 
+import PostsList from './components/PostsList/PostsList';
+
+function getPostWithProps(postsArr, usersArr, commentsArr) {
+
```

Comment:
remove this blank line, there is no need for a blank line when you just open brackets